### PR TITLE
Fix robot images

### LINKED ISSUE #403: Add picture of robots in front-end
**Describe the improvement you would like to see**
Add picture of the robots under 'Robot status' in front-end.

**How will this change existing functionality?**
Show a picture of each robot.
image

**How will this improvement affect the current Threat Model?**
Does not affect.

### PR DESCRIPTION
**Pull request by aestene**
Closes #403 

image

### CODE REVIEW

File: backend/api/Database/Models/RobotModel.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+{
+    public enum RobotModel
+    {
+        TaurobInspector,
+        TaurobOperator,
+        ExR2,
+        Turtlebot,
+        AnymalX,
+        AnymalD,
+    }
```
Comment (Christdej) [inline backend/api/Database/Models/RobotModel.cs:11]: Should we add a "default" or generic "robot" case? If we want to test something or use the isar-robot repo which is none of these robots.

**Review by Christdej — CHANGES_REQUESTED** [review]

File: backend/api/Database/Models/RobotModel.cs (hunk 1 above)
Comment (aestene) [reply backend/api/Database/Models/RobotModel.cs:11]: I decided to use the `Turtlebot` as the default since that is our simulator and the one used for testing rather than have a default name `Robot` or similar.

Comment (aestene) [reply backend/api/Database/Models/RobotModel.cs:11]: So `Turtlebot` is used in mocks and tests etc.

Comment (Christdej) [reply backend/api/Database/Models/RobotModel.cs:11]: Yeah, agreed

**Review by Christdej — CHANGES_REQUESTED** [review]
Make a commit with the DB migrations in same PR?

**Review by Christdej — APPROVED** [review]
